includes/class-fs-plugin-updater.php

```diff
@@ -1188,7 +1190,7 @@ private static function get_tested_wp_version( $tested_up_to ) {
          * @return object
          */
         private function get_latest_download_details( $addon_id = false, $newer_than = false, $fetch_readme = true ) {
-            return $this->_fs->_fetch_latest_version( $addon_id, true, WP_FS__TIME_24_HOURS_IN_SEC, $newer_than, $fetch_readme );
+            return $this->_fs->_fetch_latest_version( $addon_id, true, FS_Plugin_Updater::UPDATES_CHECK_CACHE_EXPIRATION, $newer_than, $fetch_readme );

```

Comment:
Only here it seems the cache time is changed from 24hrs to 1hr. Please let me know if this is intentional.